test/unit/common/CommandLineUtils.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { Messages } from '@salesforce/core';
+import { expect } from 'chai';
+import { FlagsConfigType } from '../../../src/common/Common.js';
+import { CommandLineUtils } from '../../../src/common/CommandLineUtils.js';
+
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+
+describe('Commons utils tests', () => {
```

Comment:
Simply moved these test from `Common.test.ts` b/c the class `CommandLineUtils` was moved out of `Common.ts`